scripts/commit_checker.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -ex
+#
+# Copyright (C) 2020 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+#
+
+COMMIT_MESSAGE="`git log -3`"
+
+echo "${COMMIT_MESSAGE}"
+
+
+for line in {5..20}
```

Comment:
These rules should be tested:
1. Summary (1st line) is less the 50 chars wide.
2. Extra newline between summary and detailed text.
3. The detailed text has more than one line.
4. Each line of detailed text is less than 72-80 chars.
5. An extra new line between detailed text and the next section (bullet points or ticket number)
6. Each commit contains a ticket number and one of "Resolves: " || "Relates-to: " (note the extra space after ":".
7. If multiple tickets are referenced they should be separated by "," and a space, 
    e.g. `Resolves: OLPEDGE-123, OLPEDGE-124`

Reply:
met all rules except 5 and 7 as not mandatory for now

Reply:
Fifty chars for the subject is too small. It doesn't fit a reasonably long sentence as `Use OnlineIfNotFound to create the cache in integration tests`.